numpy/lib/tests/test_nanfunctions.py

```diff
@@ -22,6 +22,18 @@
          np.array([0.1042, -0.5954]),
          np.array([0.1610, 0.1859, 0.3146])]
 
+# Rows of _ndat with nans converted to ones
+_rdat_ones = [np.array([0.6244 , 1.0     , 0.2692 , 0.0116 , 1.0    , 0.1170])  ,
```

Comment:
This is technically a pep8 spacing violation, so it's probably worth correcting.

I know this is a little pedantic, but lots of developers setup their editors to run lint automatically, which I recommend also adopting. In pandas, violating PEP8 now causes a test suite error but in NumPy we still rely on manual checks.